packages/javascript-sdk/src/config/interfaces.ts

```diff
@@ -33,6 +33,7 @@ interface ConfigOptions {
   tokenStore?: TokenStoreObject | 'indexedDB' | 'sessionStorage' | 'localStorage';
   tree?: string;
   type?: string;
+  oauthThreshold?: number;
```

Comment:
Yes I agree with you here, but I went with consistency with the iOS and Android SDKs - they both call it **forgerock_oauth_threshold** in the config. If you or @ryanbas21 feel strongly that we should go with something more descriptive then I'm happy to go with that, otherwise we can keep it somewhat consistent